skunk.gemspec

```diff
@@ -39,7 +39,6 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "rubycritic", ">= 4.5.2", "< 5.0"
-  spec.add_dependency "terminal-table", "~> 3.0"
```

Comment:
I think we have to keep the dependencies in this file or it won't be installed when installing the gem (when installing the gem I understand it doesn't check the Gemfile) and they won't show up in rubygems

Reply:
I'll check it then

Reply:
yeah, good point, we do require `terminal-table` as a dependency because it's used to generate the console report. So removing it from here will cause problems...

Reply:
Good catch! Moved back to the skunk.gemspec file